### PR TITLE
[nifi] NiFi helm chart

### PR DESCRIPTION
**Pull request by Hsankesara**
**Description of the change**

Since [cetic helm-nifi](https://github.com/cetic/helm-nifi) is not being maintained any more, and we are planning to integrate the NiFI in the stack, it would be good to maintain NiFi helm charts. I have copied the helm chart from the [cetic/helm-nifi](https://github.com/cetic/helm-nifi) repo. In future, I will work on updating the nifi version to NiFi 2.0.0. 

**Checklist**

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. [<name_of_the_chart>])

### CODE REVIEW

**Comment by yatharthranjan** [comment]
Hi, Thanks for this. 

The CI is failing, you need to add a line for the new chart in the Makefile, similar to other charts with `update-*`. See previous PRs that added new external charts for reference.

Keeping a copy of the chart is a good idea, but I would not recommend using an archived (unmaintained) chart, as the updates and maintenance can be laborious. Instead, try finding alternative charts that are maintained, so when an update is made, you can just copy the update here, reducing the overheads. This is fine for now, but let's add an issue to update it in the future.
Some alternatives:

1. https://artifacthub.io/packages/helm/d4n/nifi
2. There is also a Kubernetes operator, but probably keep that for later - https://konpyutaika.github.io/nifikop/docs/1_concepts/1_start_here

**Comment by Hsankesara** [comment]
Hi @yatharthranjan, 

I'll work on the CI changes. I think the issue here is I cannot find any NiFi hem chart repo that some institution maintains. [aristide/nifi](https://github.com/aristide/helm-repo/tree/main/charts/nifi) seems like it's been maintained by an individual. I'm not sure how consistent it's going to be for future updates. It does have nifi-2.0, so I would have a look.

File: Makefile [hunk 1]
```diff
@@ -124,3 +124,10 @@ update-velero:
 	@helm repo add $(patsubst update-%,%,$@) https://vmware-tanzu.github.io/helm-charts
 	@helm pull -d external --untar $(patsubst update-%,%,$@)/$(patsubst update-%,%,$@)
 	@echo ""
+
+update-nifi:
+	@echo "Updating NiFi"
+	@rm -rf external/$(patsubst update-%,%,$@)
+	@helm repo add $(patsubst update-%,%,$@) https://artifacthub.io/packages/helm/cetic/nifi
```
Comment (keyvaann) [inline Makefile:131]: I think this should be https://cetic.github.io/helm-charts

Comment (Hsankesara) [reply Makefile:131]: Okay sure, I was just confused because the repo isn't being maintained anymore.

**Review by keyvaann — APPROVED** [review]
LGTM

**Pull request merged**